components/Companies.tsx

```diff
@@ -1,53 +1,44 @@
 import { companies } from "../data/companies";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faLocationDot } from "@fortawesome/free-solid-svg-icons";
+import CareerLink from "./CareerLink";
+import WebsiteLink from "./WebsiteLink";
+import Company from "./Company";
 
 export default function Companies() {
+  const Tag = ({ children }) => {
+    return (
+      <div className="bg-stone-900 rounded-md px-2 py-[2px] mr-2 min-w-max">
+        <p className="font-thin text-xs text-gray-400 text-center">
+          {children}
+        </p>
+      </div>
+    );
+  };
   return (
-    <div className="">
+    <div className="flex flex-col w-full sm:grid sm:grid-cols-2 sm:gap-4 sm:content-start lg:grid-cols-3">
       {companies.map((company) => (
-        <div
-          className="rounded-lg shadow-lg text-center outline-white outline-1 outline w-1/3"
-          key={company.name}
-        >
-          <div className="py-3 px-6 border-b border-white">
-            Consulting Firm?
+        <Company key={company.name}>
+          <h2 className="text-xl font-medium">{company.name}</h2>
```

Comment:
I think "Local Tech Companies" is already being used as an H2, so maybe this should be an h3?